src/commands/submit.js

```diff
@@ -288,6 +288,18 @@ const submitTask = async () => {
         checkSolution(result, solution, exercises.length);
       });
     });
+  } else {
+    exec(`dart run ${submittedFile}`, (err, result) => {
+      if (err) {
+        logger.error(' Oops there is something wrong with the syntax part!');
+        process.exit(1);
+      }
+      exec(`dart run ${solutionFile}`, (err, solution) => {
+        if (err) throw err;
+        validateSolution(submittedFile);
+        checkSolution(result, solution, exercises.length);
+      });
+    });
```

Comment:
Guess this if else block can be refactored by a switch case and can combine the general steps into a function and be invoked depending on each case 
cc: @jamesgeorge007

Reply:
@ajomadlabs can I work on that?

Reply:
@shan-shaji Yes you can go ahead and make the required changes